pkg/broker/broker.go

```diff
@@ -959,3 +965,11 @@ func ocLogin(log *logging.Logger, args ...string) error {
 	}
 	return nil
 }
+
+func extractNameFromPlanID(planID string) (string, error) {
+	splitPlan := strings.Split(planID, "-")
+
+	// TOOD: Error Handling
```

Comment:
Can planID be just '-'? If neither of these cases can occur or are error checked elsewhere then you don't need any error checking. If they aren't, then they'll need to be checked because they will cause runtime errors.